### DISCUSSION TITLE
Connecting Trino to Databricks on Azure

### DISCUSSION THREAD

**Discussion by malcolndandaro** [trinodb/trino#16814] [Q&A]
Hello, i'm trying to setup Trino to connect to our Databricks workspace that is running on Azure. I wasn't able to connect till now, and didn't find much documentation about it.

Our workspace uses a external hive database (mariadb) but we don't have a separated thrift server, we use the[ "local mode" option](https://docs.databricks.com/data/metastores/external-hive-metastore.html)

I tried to do some test with the "trino-delta-lake" connector but he asks for a hive.metastore.uri, which i don't think i have.
Has anyone been able to do this connection? or i actually need to use a "remote mode" thrift server in our workspace?

Is there anyway to connect to the JDBC endpoint that the cluster has? like `jdbc:databricks://adb-xxxxxxxxx.azuredatabricks.net:443/....`

**Comment by malcolndandaro** [comment]
Just a update: I was able to connect by deploying a Hive metastore server using this docker image (https://github.com/naushadh/hive-metastore) and connecting this server to the metastore DB i already use. 

But i still would like to know if theres a way to connect to the cluster JDBC endpoint, just like we do with SQL Server and other databases.

**Reply by ebyhr** [reply]
> But i still would like to know if theres a way to connect to the cluster JDBC endpoint

JDBC connection isn't supported and there's no plan to support it.
